application/application-Equitable-Origin.md

```diff
@@ -0,0 +1,191 @@
+
+## 1.4. Governing Authority 
+
+1)	Equitable Origin is the Governing Authority responsible for this Governance Framework (GF).
+3)	Company Registration Information - [Equitable Origin]
+4)	The contact information for [Equitable Origin](https://www.equitableorigin.org/) during the pilot phase of development is:
+* 	**Name:** 
+* 	**Title:** 
+* 	**Organization:** Equitable Origin
+* 	**Email:** 
+
+## 1.5. Administering Authority
+
+[Energy, Mines and Digital Trust (EMDT)](https://digital.gov.bc.ca/case-studies/emdt) is the Administering Authority on behalf of Equitable Origin during the pilot phase of development.
+
+## 1.6. Purpose
+
+The purpose of this governance framework is to describe the rules/policies/procedures for verifiable credential exchanges involving Equitable Origin Services Inc. with the open global community. The purpose of the rules is to enable all actors to understand agreed upon standards, terminology and processes that allow the community to interact with Equitable Origin in a trusted manner. This will help determine a governing framework and operating model for a global ecosystem that identifies how credentials can be issued, held, and verified. 
+
+## 1.7. Scope 
+
+Equitable Origin is a participant in an open ecosystem and the focus of this framework is to describe the process Equitable Origin uses for digital exchanges on Indy networks.
+
+## 1.8. Objectives
+
+_This section states the high-level outcomes desired by the trust community through its adoption of the GF._
+
+1. SHOULD specify tangible, achievable results (e.g. SMART criteria and Fit-for-Purpose criteria).
+2. MUST only contain outcomes over which the GF has the authority and mechanisms to achieve within its
+scope.
+3. MUST be consistent with the principles of the GF (below).
+
+
+## 1.9. Principles 
+
+_these are standard principals from ToIP, example to be edited by Equitable Origin_
+1)	Sustainability
+2)	Stewardship 
+3)	Public good
+4)	Transparency
+5)	Citizen Engagement
+6)	Digital Sovereignty
+7)	Security
+8)	Confidence
+9)	Synchronization
+
+## 1.10. General Requirements
+TBD
+
+## 1.11. Revisions
+Version 1.0
+_TBD - How and when Equitable Origin would change governance framework_
+
+## 1.12. Extensions
+
+There are no extensions to this Governance Framework.
+
+## 1.13. Schedule of Controlled Documents
+
+TBD
+
+# 2. Controlled Documents
+
+## 2.1. Glossary
+N/A
+
+## 2.2. Risk Assessment
+
+TBD
+
+## 2.3. Trust Assurance and Certification
+
+TBD
+
+## 2.4. Governance Requirements
+
+TBD
+
+## 2.5. Business Requirements
+
+1. Equitable Origin MUST send an invitation to the other business entity via email to initiate the exchange of information. 
+2. The receiving party MUST accept invitation in order to establish a secure connection.
+
+** Issuance of Credential **
+
+1. Using the established connection, Equitable Origin MUST receive a credential request from the business entity. 
```

Comment:
Let's be more specific by referencing the EO100 credential (with hyperlink to the EO100 Credential GF) and talk about it in the context of the specific business process. Something like, "Processing a Request for an EO100 Certification".